Domain.Sql/EfDbConfiguration.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Data.Entity;
+using System.Data.Entity.SqlServer;
+
+namespace Microsoft.Its.Domain.Sql
+{
+    public class EfDbConfiguration : DbConfiguration
+    {

```

Comment:
Based this msdn document, https://msdn.microsoft.com/en-us/data/jj680699.aspx, 

Code-based configuration in EF6 and above is achieved by creating a subclass of System.Data.Entity.Config.DbConfiguration. The following guidelines should be followed when subclassing DbConfiguration:

Create only one DbConfiguration class for your application. This class specifies app-domain wide settings.
Place your DbConfiguration class in the same assembly as your DbContext class. (See the Moving DbConfiguration section if you want to change this.)
Give your DbConfiguration class a public parameterless constructor.
Set configuration options by calling protected DbConfiguration methods from within this constructor.
Following these guidelines allows EF to discover and use your configuration automatically by both tooling that needs to access your model and when your application is run.